.env.dist

```diff
@@ -1 +1 @@
-DAPPRADAR_API_KEY=your-sample-key-from-dappradar
+DAPPRADAR_API_KEY=your-sample-key-from-dappradar
```

Comment:
Please revert this file's changes. Nothing changed here.

Reply:
the first commit deleted this file. I just restored it